task/07-yield-tasks.js

```diff
@@ -126,7 +159,15 @@ function* breadthTraversalTree(root) {
  *   [ 1, 3, 5, ... ], [ -1 ] => [ -1, 1, 3, 5, ...]
  */
 function* mergeSortedSequences(source1, source2) {
-    throw new Error('Not implemented');
+    let sour1 = source1();
+    let sour2 = source2();
+    while (true) {
+        let s1 = sour1.next();
+        let s2 = sour2.next();
+        if (s1.done) yield s2.value;
+        else if(s2.done) yield s1.value;
+        else yield * [s1.value, s2.value].sort((a,b) => {return a-b});
```

Comment:
1) use simplified version of the arrow function
2) this is not going to work if you need to grab several elements from s1 and then a few from s2